R/round.r

```diff
@@ -159,10 +161,10 @@ parse_unit_spec <- function(unitspec) {
     mult <- as.numeric(parts[[1]])
     unit <- parts[[2]]
   }
-  
+
```

Comment:
I was looking at one your PRs and Hadley mentioned cleaning up the diffs to not show stuff like this. How do you remove diffs like this which appear from adding/removing newline characters?